src/codecovio.jl

```diff
@@ -62,18 +62,42 @@ module Codecov
 
     Takes a vector of file coverage results (produced by `process_folder`),
     and submits them to Codecov.io. Assumes that this code is being run
-    on TravisCI. If running locally, use `submit_local`.
+    on TravisCI or AppVeyor. If running locally, use `submit_local`.
     """
     function submit(fcs::Vector{FileCoverage}; kwargs...)
-        kwargs = set_defaults(kwargs,
-            service      = "travis-org",
-            branch       = ENV["TRAVIS_BRANCH"],
-            commit       = ENV["TRAVIS_COMMIT"],
-            pull_request = ENV["TRAVIS_PULL_REQUEST"],
-            job          = ENV["TRAVIS_JOB_ID"],
-            slug         = ENV["TRAVIS_REPO_SLUG"],
-            build        = ENV["TRAVIS_JOB_NUMBER"],
+        if haskey(ENV, "APPVEYOR") && lowercase(ENV["APPVEYOR"]) == "true"
+            appveyor_pr = haskey(ENV, "APPVEYOR_PULL_REQUEST_NUMBER") ?
```

Comment:
Ah I forgot about that, it makes perfect sense for the AppVeyor PR number.

For the `ENV["TRAVIS_CI"/"APPVEYOR"]` ones, how does this look?

```if lowercase(get(ENV, $the_ci_name", "false")) == "true"```